Unix/http/httpauth.c

```diff
@@ -473,6 +476,7 @@ MI_Boolean Http_EncryptData(_In_ Http_SR_SocketData * handler, _Out_ char **pHea
 
     gss_buffer_desc token = { 0 };
 
+#if 0
```

Comment:
I still see this in your latest branch on the server. Please push a new branch so I can see the changes, thanks.